app/decorators/module_overview_decorator.rb

```diff
@@ -92,7 +92,7 @@ def heading(page)
   # @param pages [Array<Training::Page, Training::Question, Training::Video>]
   # @return [String]
   def page_count(pages)
-    "(Pages #{pages.size})" unless pages.one?
+    "(#{pages.size} pages)" unless pages.one?
```

Comment:
Move to CMS so it's able to be changed again later?

Reply:
I have pushed a commit now that includes an update for moving this to the CMS